### PR TITLE
Uploading photos to the correct folder

### PR DESCRIPTION
**Pull request by y-lohse**
Needs this [PR](https://github.com/cozy/cozy-client/pull/241) from cozy-client, and the doctypes/references will need to be declared in only one place for drive and photos before we merge this. But the rest should be ok!

### CODE REVIEW

**Comment by y-lohse** [comment]
[(1) DOSSIER - Dossier magique "Uploaded from Cozy Photos"](https://trello.com/c/esVoDzE1/1339-1-dossier-dossier-magique-uploaded-from-cozy-photos)

**Comment by y-lohse** [comment]
This is now ready for review!

File: src/drive/web/modules/navigation/RealtimeFiles.jsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ class RealtimeFiles extends React.Component {
   pouchListener = null
 
   async componentWillMount() {
-    const { client } = this.context.client
+    const { stackClient: client } = this.context.client
```
Comment (Crash--) [inline src/drive/web/modules/navigation/RealtimeFiles.jsx]: LOL

**Comment by y-lohse** [comment]
Yes that's a good point! I'll merge this so we can start the validation process and add some tests in a separate PR :)

**Pull request merged**